PR for V2.0

hurni opened the pull request:
changed naming of elements from "husbandry" to "animalHolding" according to feedback from native speaker.

hurni committed:
Update eCH-0262-1-0.xsd

changed naming of elements from "husbandry" to "animalHolding" according to feedback from native speaker.

AFoletti commented:
Given the branch name, this PR should act as the main V2.0 PR. That means, all the V2 changes should be merged into this branch before we merge it with master

AFoletti committed:
NonCropObjects to NonCropObject

Fixes #14

hurni committed:
change name from "appliedBy" to "utilisedBy"

"utilisedBy" is used 3x, appliedBy only once. Bot are of identical type - i.e. "eCH-0108:uidType"
Thus: for consistency, only use "utilisedBy" if applicable.

hurni commented:
closed PR. Merge to main from release-2.0-Branch will be done at some point in the future...
